packages/cloog.rb

```diff
@@ -1,48 +1,27 @@
-require 'package'
+require 'buildsystems/autotools'
 
-class Cloog < Package
+class Cloog < Autotools
   description 'The CLooG Code Generator in the Polytope Model'
   homepage 'https://github.com/periscop/cloog'
-  version '0.20.0'
+  version '0.21.1'
   license 'LGPL-2.1'
   compatibility 'all'
-  source_url 'https://github.com/periscop/cloog/releases/download/cloog-0.20.0/cloog-0.20.0.tar.gz'
-  source_sha256 '835c49951ff57be71dcceb6234d19d2cc22a3a5df84aea0a9d9760d92166fc72'
-  binary_compression 'tar.xz'
+  source_url 'https://github.com/periscop/cloog.git'
+  git_hashtag "cloog-#{version}"
+  binary_compression 'tar.zst'
 
   binary_sha256({
-    aarch64: 'b776360d2fcfcd9eaf177250f7ad3ec4c128ef62b883f0f3234d696ef45335ca',
-     armv7l: 'b776360d2fcfcd9eaf177250f7ad3ec4c128ef62b883f0f3234d696ef45335ca',
-       i686: '5c6413aa59c93f8c25c86f4642b6311ed697e7305daeef9a35a6656b5a6e0694',
-     x86_64: 'd31e2c02736dfa446b5058b4f947e2f6ed9a2aff0268adb616fedae1c716f012'
+    aarch64: '94adbe0a434c3217aea0f8e046487c8fa22f9cfae217afe5497eb5bf89ad17b0',
+     armv7l: '94adbe0a434c3217aea0f8e046487c8fa22f9cfae217afe5497eb5bf89ad17b0',
+       i686: 'edde9896159ba206e68dea645463f480b9a010b52f34a07cf2115bed27bbb919',
+     x86_64: '39e04e2dcad00ccb3a7f9dceeccb7c781320fc738c415830cdefeca3d669ca35'
   })
 
+  depends_on 'glibc' # R
+  depends_on 'gmp' # R
   depends_on 'isl'
   depends_on 'osl'
 
-  def self.patch
-    system "sed -i -e '292,293d' -e '75,76d' configure.ac"
-    system "sed -i -e '188,193d' Makefile.am"
-  end
-
-  def self.build
-    system 'rm', '-rf', 'isl', 'osl'
-    system 'mkdir', '-p', 'isl', 'osl'
-    system 'rm', '-f', 'autoconf/ltmain.sh', 'm4/ltversion.m4'
-    system 'autoreconf', '-i'
-    system './configure',
-           "--prefix=#{CREW_PREFIX}",
-           "--libdir=#{CREW_LIB_PREFIX}",
-           '--with-isl=system',
-           '--with-osl=system'
-    system 'make'
-  end
-
-  def self.install
-    system 'make', 'install', "DESTDIR=#{CREW_DEST_DIR}"
-  end
-
-  def self.check
-    system 'make', 'check'
-  end
+  configure_options '--with-isl=system --with-osl=system'
+  run_tests
```

Comment:
No filelist updates for cloog?

Reply:
no filelist updates, as mentioned in the pr description.